lib/ts/recipe/session/types.ts

```diff
@@ -410,7 +412,10 @@ export type APIInterface = {
      */
     refreshPOST:
         | undefined
-        | ((input: { options: APIOptions; userContext: UserContext }) => Promise<SessionContainerInterface>);
+        | ((input: {
+              options: APIOptions;
+              userContext: UserContext;
+          }) => Promise<SessionContainerInterface | undefined>);
```

Comment:
I'm not sure if this isn't breaking in some cases, but it definitely doesn't feel like a patch level change.